diff --git a/adapter/network.go b/adapter/network.go
index 14fe46c..74cba54 100644
--- a/adapter/network.go
+++ b/adapter/network.go
@@ -10 +10 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/adapter/outbound.go b/adapter/outbound.go
index 031677e..631e8b1 100644
--- a/adapter/outbound.go
+++ b/adapter/outbound.go
@@ -10 +10 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/adapter/platform.go b/adapter/platform.go
index df1f447..b3e58c7 100644
--- a/adapter/platform.go
+++ b/adapter/platform.go
@@ -7 +7 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/adapter/router.go b/adapter/router.go
index 3d5310c..2b19d35 100644
--- a/adapter/router.go
+++ b/adapter/router.go
@@ -12 +12 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/cmd/sing-box/cmd_check.go b/cmd/sing-box/cmd_check.go
index 29a3908..d8b7e63 100644
--- a/cmd/sing-box/cmd_check.go
+++ b/cmd/sing-box/cmd_check.go
@@ -6 +6 @@ import (
-	"github.com/sagernet/sing-box"
+	box "github.com/sagernet/sing-box"
diff --git a/cmd/sing-box/cmd_run.go b/cmd/sing-box/cmd_run.go
index f31db9d..68f8713 100644
--- a/cmd/sing-box/cmd_run.go
+++ b/cmd/sing-box/cmd_run.go
@@ -15 +15 @@ import (
-	"github.com/sagernet/sing-box"
+	box "github.com/sagernet/sing-box"
diff --git a/cmd/sing-box/cmd_tools.go b/cmd/sing-box/cmd_tools.go
index 5c6a112..d028f87 100644
--- a/cmd/sing-box/cmd_tools.go
+++ b/cmd/sing-box/cmd_tools.go
@@ -9 +9 @@ import (
-	"github.com/sagernet/sing-box"
+	box "github.com/sagernet/sing-box"
diff --git a/common/badtls/registry_utls.go b/common/badtls/registry_utls.go
index 330f64f..c5d7aed 100644
--- a/common/badtls/registry_utls.go
+++ b/common/badtls/registry_utls.go
@@ -11 +11 @@ import (
-	"github.com/metacubex/utls"
+	tls "github.com/metacubex/utls"
diff --git a/common/mux/client.go b/common/mux/client.go
index bf10335..0593693 100644
--- a/common/mux/client.go
+++ b/common/mux/client.go
@@ -10 +10 @@ import (
-	"github.com/sagernet/sing-mux"
+	mux "github.com/sagernet/sing-mux"
diff --git a/common/mux/router.go b/common/mux/router.go
index ec78808..5b8a077 100644
--- a/common/mux/router.go
+++ b/common/mux/router.go
@@ -11 +11 @@ import (
-	"github.com/sagernet/sing-mux"
+	mux "github.com/sagernet/sing-mux"
diff --git a/common/process/searcher.go b/common/process/searcher.go
index 6430523..b5039c5 100644
--- a/common/process/searcher.go
+++ b/common/process/searcher.go
@@ -10 +10 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/common/process/searcher_android.go b/common/process/searcher_android.go
index e634774..96ad83e 100644
--- a/common/process/searcher_android.go
+++ b/common/process/searcher_android.go
@@ -8 +8 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/common/settings/proxy_darwin.go b/common/settings/proxy_darwin.go
index baaf6ce..e894ccb 100644
--- a/common/settings/proxy_darwin.go
+++ b/common/settings/proxy_darwin.go
@@ -9 +9 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/common/tls/std_client.go b/common/tls/std_client.go
index 56b9e61..5bbd3be 100644
--- a/common/tls/std_client.go
+++ b/common/tls/std_client.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-box/common/tlsfragment"
+	tf "github.com/sagernet/sing-box/common/tlsfragment"
diff --git a/common/tls/utls_client.go b/common/tls/utls_client.go
index 1db3e49..9a337e5 100644
--- a/common/tls/utls_client.go
+++ b/common/tls/utls_client.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-box/common/tlsfragment"
+	tf "github.com/sagernet/sing-box/common/tlsfragment"
diff --git a/common/tlsfragment/index_test.go b/common/tlsfragment/index_test.go
index 5086d6c..63dc758 100644
--- a/common/tlsfragment/index_test.go
+++ b/common/tlsfragment/index_test.go
@@ -7 +7 @@ import (
-	"github.com/sagernet/sing-box/common/tlsfragment"
+	tf "github.com/sagernet/sing-box/common/tlsfragment"
diff --git a/daemon/instance.go b/daemon/instance.go
index 3acf75c..ad4c174 100644
--- a/daemon/instance.go
+++ b/daemon/instance.go
@@ -7 +7 @@ import (
-	"github.com/sagernet/sing-box"
+	box "github.com/sagernet/sing-box"
diff --git a/dns/router.go b/dns/router.go
index 4f18959..317fcf4 100644
--- a/dns/router.go
+++ b/dns/router.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/dns/transport/dhcp/dhcp.go b/dns/transport/dhcp/dhcp.go
index f9702da..956b7af 100644
--- a/dns/transport/dhcp/dhcp.go
+++ b/dns/transport/dhcp/dhcp.go
@@ -19 +19 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/dns/transport/local/local_resolved_linux.go b/dns/transport/local/local_resolved_linux.go
index fc3ca2b..99ec5e2 100644
--- a/dns/transport/local/local_resolved_linux.go
+++ b/dns/transport/local/local_resolved_linux.go
@@ -21 +21 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/include/registry.go b/include/registry.go
index 9d45376..11587e4 100644
--- a/include/registry.go
+++ b/include/registry.go
@@ -6 +6 @@ import (
-	"github.com/sagernet/sing-box"
+	box "github.com/sagernet/sing-box"
diff --git a/protocol/direct/inbound.go b/protocol/direct/inbound.go
index 81353b6..eebc464 100644
--- a/protocol/direct/inbound.go
+++ b/protocol/direct/inbound.go
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing/common/udpnat2"
+	udpnat "github.com/sagernet/sing/common/udpnat2"
diff --git a/protocol/direct/outbound.go b/protocol/direct/outbound.go
index e483339..b875947 100644
--- a/protocol/direct/outbound.go
+++ b/protocol/direct/outbound.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/protocol/group/urltest.go b/protocol/group/urltest.go
index 730040f..311ea0c 100644
--- a/protocol/group/urltest.go
+++ b/protocol/group/urltest.go
@@ -17 +17 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/protocol/naive/quic/inbound_init.go b/protocol/naive/quic/inbound_init.go
index b391a26..0bed818 100644
--- a/protocol/naive/quic/inbound_init.go
+++ b/protocol/naive/quic/inbound_init.go
@@ -17 +17 @@ import (
-	"github.com/sagernet/sing-quic"
+	qtls "github.com/sagernet/sing-quic"
diff --git a/protocol/redirect/tproxy.go b/protocol/redirect/tproxy.go
index f0b82bb..fea420f 100644
--- a/protocol/redirect/tproxy.go
+++ b/protocol/redirect/tproxy.go
@@ -21 +21 @@ import (
-	"github.com/sagernet/sing/common/udpnat2"
+	udpnat "github.com/sagernet/sing/common/udpnat2"
diff --git a/protocol/shadowsocks/inbound.go b/protocol/shadowsocks/inbound.go
index 52e2c52..09086d1 100644
--- a/protocol/shadowsocks/inbound.go
+++ b/protocol/shadowsocks/inbound.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-shadowsocks"
+	shadowsocks "github.com/sagernet/sing-shadowsocks"
diff --git a/protocol/shadowsocks/inbound_multi.go b/protocol/shadowsocks/inbound_multi.go
index 7ff9264..ef0d87f 100644
--- a/protocol/shadowsocks/inbound_multi.go
+++ b/protocol/shadowsocks/inbound_multi.go
@@ -17 +17 @@ import (
-	"github.com/sagernet/sing-shadowsocks"
+	shadowsocks "github.com/sagernet/sing-shadowsocks"
diff --git a/protocol/shadowsocks/outbound.go b/protocol/shadowsocks/outbound.go
index ebf21f6..ed4082b 100644
--- a/protocol/shadowsocks/outbound.go
+++ b/protocol/shadowsocks/outbound.go
@@ -15 +15 @@ import (
-	"github.com/sagernet/sing-shadowsocks2"
+	shadowsocks "github.com/sagernet/sing-shadowsocks2"
diff --git a/protocol/shadowtls/inbound.go b/protocol/shadowtls/inbound.go
index 17afa26..7780516 100644
--- a/protocol/shadowtls/inbound.go
+++ b/protocol/shadowtls/inbound.go
@@ -14 +14 @@ import (
-	"github.com/sagernet/sing-shadowtls"
+	shadowtls "github.com/sagernet/sing-shadowtls"
diff --git a/protocol/shadowtls/outbound.go b/protocol/shadowtls/outbound.go
index 41a4a60..f4b59a6 100644
--- a/protocol/shadowtls/outbound.go
+++ b/protocol/shadowtls/outbound.go
@@ -15 +15 @@ import (
-	"github.com/sagernet/sing-shadowtls"
+	shadowtls "github.com/sagernet/sing-shadowtls"
diff --git a/protocol/tailscale/endpoint.go b/protocol/tailscale/endpoint.go
index 03ec6c9..08de40f 100644
--- a/protocol/tailscale/endpoint.go
+++ b/protocol/tailscale/endpoint.go
@@ -34 +34 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/protocol/tun/inbound.go b/protocol/tun/inbound.go
index 8937fbe..15f6d2c 100644
--- a/protocol/tun/inbound.go
+++ b/protocol/tun/inbound.go
@@ -20 +20 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/protocol/vmess/inbound.go b/protocol/vmess/inbound.go
index 4e9c763..3a1630f 100644
--- a/protocol/vmess/inbound.go
+++ b/protocol/vmess/inbound.go
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing-vmess"
+	vmess "github.com/sagernet/sing-vmess"
diff --git a/protocol/vmess/outbound.go b/protocol/vmess/outbound.go
index 1baf3b3..cac926b 100644
--- a/protocol/vmess/outbound.go
+++ b/protocol/vmess/outbound.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-vmess"
+	vmess "github.com/sagernet/sing-vmess"
diff --git a/protocol/wireguard/endpoint.go b/protocol/wireguard/endpoint.go
index 63009bf..b2d057a 100644
--- a/protocol/wireguard/endpoint.go
+++ b/protocol/wireguard/endpoint.go
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/route/conn.go b/route/conn.go
index 9fdc6cd..8a515e8 100644
--- a/route/conn.go
+++ b/route/conn.go
@@ -17 +17 @@ import (
-	"github.com/sagernet/sing-box/common/tlsfragment"
+	tf "github.com/sagernet/sing-box/common/tlsfragment"
diff --git a/route/dns.go b/route/dns.go
index dee4a75..146d083 100644
--- a/route/dns.go
+++ b/route/dns.go
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing/common/udpnat2"
+	udpnat "github.com/sagernet/sing/common/udpnat2"
diff --git a/route/network.go b/route/network.go
index da79a8e..4ec4224 100644
--- a/route/network.go
+++ b/route/network.go
@@ -20 +20 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/route/route.go b/route/route.go
index 5fadff0..554c38a 100644
--- a/route/route.go
+++ b/route/route.go
@@ -15,2 +15,2 @@ import (
-	"github.com/sagernet/sing-mux"
-	"github.com/sagernet/sing-tun"
+	mux "github.com/sagernet/sing-mux"
+	tun "github.com/sagernet/sing-tun"
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing-vmess"
+	vmess "github.com/sagernet/sing-vmess"
diff --git a/route/rule/rule_action.go b/route/rule/rule_action.go
index cac814e..5ab4f3b 100644
--- a/route/rule/rule_action.go
+++ b/route/rule/rule_action.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/route/rule/rule_default_interface_address.go b/route/rule/rule_default_interface_address.go
index 2d7fdeb..1009f77 100644
--- a/route/rule/rule_default_interface_address.go
+++ b/route/rule/rule_default_interface_address.go
@@ -8 +8 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/test/box_test.go b/test/box_test.go
index d7d9b9b..2930e09 100644
--- a/test/box_test.go
+++ b/test/box_test.go
@@ -14 +14 @@ import (
-	"github.com/sagernet/sing-box"
+	box "github.com/sagernet/sing-box"
diff --git a/transport/sip003/obfs.go b/transport/sip003/obfs.go
index 129e775..21eec18 100644
--- a/transport/sip003/obfs.go
+++ b/transport/sip003/obfs.go
@@ -8 +8 @@ import (
-	"github.com/sagernet/sing-box/transport/simple-obfs"
+	obfs "github.com/sagernet/sing-box/transport/simple-obfs"
diff --git a/transport/sip003/v2ray.go b/transport/sip003/v2ray.go
index f35e265..d72eca5 100644
--- a/transport/sip003/v2ray.go
+++ b/transport/sip003/v2ray.go
@@ -13 +13 @@ import (
-	"github.com/sagernet/sing-vmess"
+	vmess "github.com/sagernet/sing-vmess"
diff --git a/transport/v2rayquic/client.go b/transport/v2rayquic/client.go
index c4e0b10..2929fbc 100644
--- a/transport/v2rayquic/client.go
+++ b/transport/v2rayquic/client.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-quic"
+	qtls "github.com/sagernet/sing-quic"
diff --git a/transport/v2rayquic/server.go b/transport/v2rayquic/server.go
index c50d92f..6fd1ad8 100644
--- a/transport/v2rayquic/server.go
+++ b/transport/v2rayquic/server.go
@@ -16 +16 @@ import (
-	"github.com/sagernet/sing-quic"
+	qtls "github.com/sagernet/sing-quic"
diff --git a/transport/wireguard/device.go b/transport/wireguard/device.go
index 81d610e..d141193 100644
--- a/transport/wireguard/device.go
+++ b/transport/wireguard/device.go
@@ -9 +9 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/device_nat.go b/transport/wireguard/device_nat.go
index d214b73..495fb8a 100644
--- a/transport/wireguard/device_nat.go
+++ b/transport/wireguard/device_nat.go
@@ -10 +10 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/device_stack.go b/transport/wireguard/device_stack.go
index 3408850..1b67ac0 100644
--- a/transport/wireguard/device_stack.go
+++ b/transport/wireguard/device_stack.go
@@ -25 +25 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/device_stack_gonet.go b/transport/wireguard/device_stack_gonet.go
index bc69520..66d49a0 100644
--- a/transport/wireguard/device_stack_gonet.go
+++ b/transport/wireguard/device_stack_gonet.go
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/device_stack_stub.go b/transport/wireguard/device_stack_stub.go
index ea41355..7c90d33 100644
--- a/transport/wireguard/device_stack_stub.go
+++ b/transport/wireguard/device_stack_stub.go
@@ -5 +5 @@ package wireguard
-import "github.com/sagernet/sing-tun"
+import tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/device_system.go b/transport/wireguard/device_system.go
index 1c0b8b6..e3e4181 100644
--- a/transport/wireguard/device_system.go
+++ b/transport/wireguard/device_system.go
@@ -13 +13 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/device_system_stack.go b/transport/wireguard/device_system_stack.go
index 1172e60..cdcbc06 100644
--- a/transport/wireguard/device_system_stack.go
+++ b/transport/wireguard/device_system_stack.go
@@ -22 +22 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/endpoint.go b/transport/wireguard/endpoint.go
index c868258..9390b22 100644
--- a/transport/wireguard/endpoint.go
+++ b/transport/wireguard/endpoint.go
@@ -18 +18 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
diff --git a/transport/wireguard/endpoint_options.go b/transport/wireguard/endpoint_options.go
index 1f950ef..3900be5 100644
--- a/transport/wireguard/endpoint_options.go
+++ b/transport/wireguard/endpoint_options.go
@@ -8 +8 @@ import (
-	"github.com/sagernet/sing-tun"
+	tun "github.com/sagernet/sing-tun"
